src/main/webapp/jsp/instructorFeedbackResultsByGiverRecipientQuestion.jsp

```diff
@@ -470,9 +471,14 @@
                             <div class="col-md-10">
                     <%
                     	int qnIndx = 1;
+                       boolean isQnCONTRIB = false; /* is the Question a Contribution Question? */
```

Comment:
can make better naming? refer to our java coding standards.
thus can remove its comment